experimenter/experimenter/experiments/models.py

```diff
@@ -127,106 +128,162 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         choices=NimbusConstants.Status.choices,
     )
     publish_status = models.CharField(
+        "Publish status",
         max_length=255,
         default=NimbusConstants.PublishStatus.IDLE,
         choices=NimbusConstants.PublishStatus.choices,
     )
     name = models.CharField(max_length=255, unique=True)
     slug = models.SlugField(max_length=NimbusConstants.MAX_SLUG_LEN, unique=True)
-    public_description = models.TextField(default="")
-    risk_mitigation_link = models.URLField(max_length=255, blank=True)
-    is_paused = models.BooleanField(default=False)
-    is_rollout_dirty = models.BooleanField(blank=False, null=False, default=False)
+    public_description = models.TextField("Experiment's description", default="")
+    risk_mitigation_link = models.URLField(
+        "Risk mitigation link", max_length=255, blank=True
+    )
+    is_paused = models.BooleanField("Is enrollment paused flag", default=False)
+    is_rollout_dirty = models.BooleanField(
+        "Approved changes flag", blank=False, null=False, default=False
+    )
     proposed_duration = models.PositiveIntegerField(
+        "Experiment's expected duration",
         default=NimbusConstants.DEFAULT_PROPOSED_DURATION,
         validators=[MaxValueValidator(NimbusConstants.MAX_DURATION)],
     )
     proposed_enrollment = models.PositiveIntegerField(
+        "Proposed enrollment duration",
         default=NimbusConstants.DEFAULT_PROPOSED_ENROLLMENT,
         validators=[MaxValueValidator(NimbusConstants.MAX_DURATION)],
     )
     population_percent = models.DecimalField[Decimal](
-        max_digits=7, decimal_places=4, default=0.0
+        "Population percent", max_digits=7, decimal_places=4, default=0.0
```

Comment:
```suggestion
        "Population Percent", max_digits=7, decimal_places=4, default=0.0
```

Use Title Case for these field names.